### PR TITLE
Port dracut arguments fix

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

(It would be nice to have also the auto-comment about where this was cherry-picked from.)

**Review by poncovka — APPROVED** [review]
Thanks!

File: tests/unit_tests/pyanaconda_tests/modules/storage/test_module_iscsi.py [hunk 1]
```diff
@@ -161,3 +161,69 @@ def test_write_configuration(self, iscsi):
...
+        iscsi.get_node.return_value = blivet_node
+        blivet_node.username = "uname"
+        blivet_node.r_username = "r_uname"
+        blivet_node.password = "passwd"
+        blivet_node.r_password = "r_passwd"
+        blivet_node.iface = "qedi.a6:26:77:80:00:63"
+        iscsi.initiator = "iqn.1994-05.com.redhat:blablabla"
+        self.assertEqual(
+            self.iscsi_interface.GetDracutArguments(Node.to_structure(self._node)),
+            ["netroot=iscsi:uname:passwd:r_uname:r_passwd@10.43.136.67::3260:qedi.a6:26:77:80:00:63:qedi.a6:26:77:80:00:63::iqn.2014-08.com.example:t1",
```
Comment (poncovka) [inline tests/unit_tests/pyanaconda_tests/modules/storage/test_module_iscsi.py:227]: It is possible to define a long string with several smaller ones:

```
[
  "This is "
  "one "
  "long string.",
  "This is another one."
]
```

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

**Comment by rvykydal** [comment]
Just updated the rhbz# reference.

**Pull request merged**